Packs/CortexResponseAndRemediation/Playbooks/playbook-WmiPrvSe.exe_Rare_Child_Command_Line.yml

```diff
@@ -0,0 +1,1365 @@
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: alert.cgoname
+            iscontext: true
+          right:
+            value:
+              simple: WmiPrvSE.exe
+          ignorecase: true
+        - operator: containsGeneral
+          left:
+            value:
+              simple: alert.cgoname
+            iscontext: true
+          right:
+            value:
+              simple: WMIC.exe
+          ignorecase: true
+        - operator: containsGeneral
+          left:
+            value:
+              simple: alert.cgoname
+            iscontext: true
+          right:
+            value:
+              simple: WMICodeCreator.exe
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1643
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: 2aabb75e-d911-4d92-8974-0891c6156934
+    type: regular
+    task:
+      id: 2aabb75e-d911-4d92-8974-0891c6156934
+      version: -1
+      name: Terminate causality process
+      description: Terminate a process tree by its causality ID. Available only for XSIAM 2.4 and above.
+      script: '|||core-terminate-causality'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "13"
+      '#none#':
+      - "27"
+    scriptarguments:
+      agent_id:
+        simple: ${alert.agentid}
+      causality_id:
+        simple: ${alert.cid}
+      timeout_in_seconds:
+        simple: "180"
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: errorPath
+    view: |-
+      {
+        "position": {
+          "x": 819,
+          "y": 1840
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "10_11_Approved": 0.38,
+      "10_16_#default#": 0.1,
+      "12_13_#error#": 0.64,
+      "17_14_#default#": 0.43,
+      "17_18_Isolate": 0.4,
+      "19_14_#default#": 0.21,
+      "19_23_Yes": 0.37,
+      "23_17_WORKSTATION": 0.46,
+      "23_24_#default#": 0.62,
+      "25_10_yes": 0.55,
+      "25_16_#default#": 0.1,
+      "28_12_WMI": 0.59,
+      "28_29_#default#": 0.6,
+      "29_13_#error#": 0.63,
+      "5_11_Malicious": 0.4,
+      "5_8_#default#": 0.42,
+      "8_10_Medium Confidence": 0.82,
+      "8_11_Malicious": 0.8,
+      "8_9_#default#": 0.64,
+      "9_10_yes": 0.25,
+      "9_25_#default#": 0.48
+    },
+    "paper": {
+      "dimensions": {
+        "height": 3387,
+        "width": 1451,
+        "x": 70,
+        "y": -150
+      }
+    }
+  }
+inputs: []
+outputs: []
+tests:
+- No tests (auto formatted)
```

Comment:
What about the silent flag? As far as I know, the playbook should first merge as silent.

Reply:
Done